### ISSUE TITLE
Replace BasicGesture with constants

### ISSUE THREAD

**Issue by pelikhan** [lancaster-university/microbit-dal#106]
The way gestures are defined might create issues in the future if an enum value is removed. Could we use #defines instead?

```
enum BasicGesture
{
    GESTURE_NONE,
    GESTURE_UP,
    GESTURE_DOWN,
    GESTURE_LEFT,
    GESTURE_RIGHT,
    GESTURE_FACE_UP,
    GESTURE_FACE_DOWN,
    GESTURE_FREEFALL,
    GESTURE_3G,
    GESTURE_6G,
    GESTURE_8G,
    GESTURE_SHAKE
};
```

**Comment by finneyj** [comment]
yep - good call.

**Comment by finneyj** [comment]
Now fixed in master. 

All references now use:
```
/**
  * Gesture events
  */
#define MICROBIT_ACCELEROMETER_EVT_NONE                     0
#define MICROBIT_ACCELEROMETER_EVT_TILT_UP                  1
#define MICROBIT_ACCELEROMETER_EVT_TILT_DOWN                2
#define MICROBIT_ACCELEROMETER_EVT_TILT_LEFT                3
#define MICROBIT_ACCELEROMETER_EVT_TILT_RIGHT               4
#define MICROBIT_ACCELEROMETER_EVT_FACE_UP                  5
#define MICROBIT_ACCELEROMETER_EVT_FACE_DOWN                6
#define MICROBIT_ACCELEROMETER_EVT_FREEFALL                 7
#define MICROBIT_ACCELEROMETER_EVT_3G                       8
#define MICROBIT_ACCELEROMETER_EVT_6G                       9
#define MICROBIT_ACCELEROMETER_EVT_8G                       10
#define MICROBIT_ACCELEROMETER_EVT_SHAKE                    11
```

Will be included in next tagged release #v2.0.0rc3
